install_files/ansible-base/roles/app/tasks/initialize_securedrop_app.yml

```diff
@@ -152,12 +152,9 @@
     - securedrop_config
 
 - name: Initialize sqlite database.
-  shell: >
-    su -s /bin/bash -c
-    "PYTHONPATH={{ securedrop_code }}
-    python -c 'import db; db.init_db()'"
-    {{ securedrop_user }}
-  when: not db.stat.exists
+  shell: './manage.py init-db --user {{ securedrop_user }}'
+  args:
+    chdir: '{{ securedrop_code }}'
```

Comment:
Just noticed on this task we lost `when: not db.stat.exists` which we want to preserve for idempotency, so let's add that back in